src/components/workers-page/workers-tab/workers-tab.component.tsx

```diff
@@ -3,20 +3,59 @@ import Table from "@material-ui/core/Table";
 import TableBody from "@material-ui/core/TableBody";
 import TableCell from "@material-ui/core/TableCell";
 import TableContainer from "@material-ui/core/TableContainer";
-import TableHead from "@material-ui/core/TableHead";
 import TableRow from "@material-ui/core/TableRow";
-import { WorkerType, WorkerTypeHelper } from "../../../common-models/worker-info.model";
+import { WorkerData, WorkerTypeHelper } from "../../../common-models/worker-info.model";
 import { useSelector } from "react-redux";
 import { ApplicationStateModel } from "../../../state/models/application-state.model";
 import { Button } from "../../common-components";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import { EnhancedTableHeaderComponent } from "./enhanced-table-header-component";
+import { ArrayHelper, Comparator, Order } from "../../../helpers/array.helper";
+import { StringHelper } from "../../../helpers/string.helper";
+import ScssVars from "../../../assets/styles/styles/custom/_variables.module.scss";
 
-interface WorkerData {
-  name: string;
-  type: WorkerType;
-  time: number;
+function descendingComparator<T>(a: T, b: T, orderBy: keyof T): Comparator {
```

Comment:
@Qwebeck masz jakiś pomysł gdzie wyrzucić kod związany z Comparatorem? Bo tworzenie helpera wydało mi się nadmiarowe na razie i nie mam na to pomysłu za bardzo.

Reply:
Myślę, że on musi być w ArrayHelperze